File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/RadioButtonsPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.GroupBox.Features
{
	public partial class RadioButtonsPage : UserControl
	{
		public RadioButtonsPage()
		{
			InitializeComponent();
		}

		private void ClassRoom_Changed(object sender, EventArgs e)
		{
            //Set Label visible if it is not
			if(!mobjLblStatusClass.Visible)
				mobjLblStatusClass.Visible = true;

            //Set text to Label using the Text property value of the RadioButton
            //that raised the event
			mobjLblStatusClass.Text = 
				string.Format("Class room: \r\n{0}", ((Control)sender).Text);
		}

		private void Requirement_Changed(object sender, EventArgs e)
		{
            //Set Label visible if it is not
			if(!mobjLblRequrements.Visible)
				mobjLblRequrements.Visible = true;

            //Set text to Label using the Text property value of the RadioButton
            //that raised the event
			mobjLblRequrements.Text = 
				string.Format("Requirements: \r\n{0}", ((Control)sender).Text);
		}
	}
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/RadioButtonsPage.vb
Namespace CompanionKit.Controls.GroupBox.Features

	Public Class RadioButtonsPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

		End Sub

        Private Sub Requirement_Changed(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjRadioButton9.CheckedChanged, mobjRadioButton8.CheckedChanged, mobjRadioButton7.CheckedChanged, mobjRadioButton6.CheckedChanged, mobjRadioButton10.CheckedChanged
            ''Set Label visible if it is not
            If Not Me.mobjLblRequrements.Visible Then
                Me.mobjLblRequrements.Visible = True
            End If
            ''Set text to Label using the Text property value of the RadioButton
            ''that raised the event
            Me.mobjLblRequrements.Text = String.Format("Requirements:" & vbCr & vbLf & "{0}", DirectCast(sender, Control).Text)

        End Sub

        Private Sub ClassRoom_Changed(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjRadioButton5.CheckedChanged, mobjRadioButton4.CheckedChanged, mobjRadioButton3.CheckedChanged, mobjRadioButton2.CheckedChanged, mobjRadioButton1.CheckedChanged
            ''Set Label visible if it is not
            If Not Me.mobjLblStatusClass.Visible Then
                Me.mobjLblStatusClass.Visible = True
            End If
            ''Set text to Label using the Text property value of the RadioBotton
            ''that raised the event
            Me.mobjLblStatusClass.Text = String.Format("Requirements: " & vbCr & vbLf & "{0}", DirectCast(sender, Control).Text)

        End Sub
    End Class

End Namespace